### DISCUSSION TITLE
ある号に含める記事の区別は、ラベルよりもマイルストーンを使いたい

### DISCUSSION THREAD

**Discussion by kakutani** [rubima/magazine.rubyist.net#433] [要望・アイデア]
(るびま編集プロセスについてのメタな話題は「要望・アイデア」とは別のカテゴリがあるとよさそう?)

* ある号のラベルはリリースされたら不要になるのに、号数が増えるとともに増えていく
* デリバリーされたら閉じることができるものだから、マイルストーンのほうがよいのでは?

…という提案です(0063は試しにその運用でやってみます)。WDYT?

ちょっと試すとIssueとPRで重複があるな〜というのが気になりましたが、作業の実態(PR)と作業の状態管理やメタデータ(Issue)と思えば2つあってもいいかなという気もするので様子を見ながらやってみます。

**Comment by neko314** [comment]
自分が中途半端な状態にしてしまった気がしてるので仕切り直すとして...ラベルは確かに微妙だと感じてるのですが、私はマイルストーン機能についてあんまり理解してないので、慣らす意味も兼ねて今進行中の0065号の管理をこの方式にしてみようかと思います。

issueとPRについては、各記事についてはPRがあれば十分、それ以外の号自体に関する話をする場として各号のissueは引き続きおいておくと良さそうに思っていています。

**Reply by neko314** [reply]
あーいや、マイルストーンの中身はissueが必要あるのか...（本当に機能について分かっていない）
